torch/testing/_internal/common_nn.py

```diff
@@ -2227,15 +2191,6 @@ def fractional_max_pool3d_test(test_case):
         skip_half=True,
         desc='complex'
     ),
-    dict(
```

Comment:
Is it actually OK to remove these? Were they only added for nondeterministic testing, and are they only used for that purpose still?

Reply:
I added most, if not all, of these just for the purpose of nondeterministic testing. But let me git blame all of them to make sure they weren't changed afterward for any other purpose.

Reply:
Confirmed, these are all safe to be deleted